runner/ansible/roles/2.2.3/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '2.2.3'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    # Check the pacemaker version IS
+    VERSION_ID=$(rpm -qv pacemaker | sed -e "s/pacemaker-\(.*\)+.*/\1/")
+    [[ $VERSION_ID -ge "{{ expected[test_id] }}" ]] && exit 0
```

Comment:
This comparison doesn't work

Reply:
Fixed, changed to use:
changed_when: config_updated.stdout is version(expected[test_id], '<')